stock_by_warehouse/models/product_product.py

```diff
@@ -77,11 +77,20 @@ def _compute_get_quantity_warehouses_json(self):
         # itself, we enable this context management
         warehouse_id = self._context.get('warehouse_id')
 
-        for warehouse in self.env['stock.warehouse'].sudo().search([]):
+        # Just in case it's asked from only a group of warehouses
+        warehouse_ids = self._context.get('warehouse_ids', False)
+        some_wh = False
```

Comment:
This variable is not necessary, is it?

Reply:
To avoid comparing if not warehouse_id and warehouse_ids in every iteration.